### PR TITLE
#50 Adds checkin_link to Rails Admin (Area)

### PR DESCRIPTION
**Pull request by hey-johnnypark**
Main changes:

- Moved checkin_link generation to Area
- Add field config for checkin_link to Area concern

closes railslove/recover-backlog#50

### CODE REVIEW

File: app/models/concerns/rails_admin_config/for_area.rb [hunk 1]
```diff
@@ -4,7 +4,9 @@ module ForArea
 
     included do
       rails_admin do
-        fields :name, :id, :company, :tickets
+        configure :checkin_link do
+        end
```
Comment (salimhb) [inline app/models/concerns/rails_admin_config/for_area.rb:8]: What does this empty block do?

**Comment by salimhb** [comment]
It looks good, but CI didn't run because of the settings for forked PRs. I reconfigured it now, please push an empty commit to trigger it again.

File: app/models/concerns/rails_admin_config/for_area.rb (hunk 1 above)
Comment (hey-johnnypark) [reply app/models/concerns/rails_admin_config/for_area.rb:8]: Hi @salimhb , 

is is about virtual fields in Rails Admin. If you do not configure it, the Admin ui fails. 

Screenshot 2020-12-18 at 09 17 14

**Comment by hey-johnnypark** [comment]
@salimhb I pushed an empty commit.

**Review by salimhb — APPROVED** [review]

**Comment by hey-johnnypark** [comment]
HI @salimhb ,

I do not have any rights on the repo to perform the merge. Could you please do it? Thank you :)

**Pull request merged**